CONTRIBUTING.md

```diff
@@ -0,0 +1,63 @@
+# Contributor Onboarding
+
+🚧 This document is still a WIP 🚧 
+
+## General
+
+ - The goal of this guide is to help you start contributing to Durable Functions
+
+## Pre-reqs
+
+ - OS
+    - Windows 10 (suggested)
+    - Mac OS X/Linux
+ - Language runtimes
+    - .NET Core 2.0
+ - Editor
+    - Visual Studio 2017 (recommended)
+    - VS Code 
```

Comment:
Is VS Code capable of building WebJobs.Extensions.DurableTask.csproj?  If not, I would suggest we only recommend VS 2017.

Reply:
Sure